openmc/weight_windows.py

```diff
@@ -1,8 +1,7 @@
 from __future__ import annotations
 from collections.abc import Iterable
 from numbers import Real, Integral
-import pathlib
-from typing import Iterable, List, Optional, Union, Dict
+from typing import Iterable, List, Optional, Dict, Sequence
```

Comment:
while I was adding ```Sequence``` to the imports I removed the unused ones (```Union``` and ```pathlib``` were unused)